CONTRIBUTING.md

```diff
@@ -24,6 +24,9 @@ PRs should follow the following guidelines
 - Should pass all system and unit tests run on GitHub Actions CI
 - Follow the code style guidelines bellow
 
+### Developer documentation
+
+We have some developer documentation (C++ classes
```

Comment:
unfinished sentence?

Reply:
I don't know what you're talking about, I think it's a perfectly